lib/utils.js

```diff
@@ -206,6 +206,12 @@ var toArrayBuffer = function (buf) {
 };
 
 var dataToType = function (data, type) {
+
+    //if payload is missing don't throw runtime exception that will block h2 connection stream execution.
+    if (data == undefined) {
```

Comment:
In JS you normally test for equality with '===' .  Double '==' is used when the condition can be truthy e.g. 1 == '1'

I can't think of a case where there is a truthy comparison with undefined?

Reply:
got it thanks! I am still trying to be expert in JS :)

Reply:
Note that `null == undefined` is `true` whereas `null === undefined` is `false`, so `==` is good if same behavior is needed when `data` is either `null` or `undefined`.

Reply:
Since we want to handle null value in the same way, hence putting `==` makes sense.